### PR TITLE
Transfer Jarvis's waiting-for styles to React

### PR DESCRIPTION
**Pull request by JustCaptcha**

### CODE REVIEW

File: universal-login-react/src/ui/styles/themes/Jarvis/waitingForThemeJarvis.sass [hunk 1]
```diff
@@ -1,10 +1,16 @@
 .unilogin-theme-jarvis.unilogin-component-waitingfor
   .unilogin-component-waitingfor-modal-pending-section
+    margin: auto
     max-width: 570px
     padding: 10px 5px 30px
 
   .unilogin-component-waitingfor-action-description
     display: none
```
Comment (JustynaBroniszewska) [inline universal-login-react/src/ui/styles/themes/Jarvis/waitingForThemeJarvis.sass:8]: Should it be here: `display:none` ?

**Comment by xitronix** [comment]
image

**Comment by xitronix** [comment]
On production: 
image

**Comment by xitronix** [comment]
On production:

image

**Comment by JustCaptcha** [comment]
image
image

**Comment by JustynaBroniszewska** [comment]
I am not sure how it looks like now. If the last screenshots are correct, then it is wrong in WaitingForDeployment component. Because we have duplication of description. So we should show only one. I suggest the one on the bottom.

**Comment by JustCaptcha** [comment]
image

**Pull request merged**